content/tutorials/jwt-authentication.textile

```diff
@@ -324,22 +365,29 @@ h2. Next steps
 7. Gain a good technical "overview of how the Ably realtime platform works":https://ably.com/docs/key-concepts
 8. Obtain complete code for this tutorial from its "GitHub repository":https://github.com/ably/jwt-auth-node-tutorial.
```

Comment:
> https://github.com/ably/jwt-auth-node-tutorial.

Wrong link to tutorial code. Also, minor point, please remove the full stop from the end of the line, as none of the other lines have it.

Reply:
- the link is correct (merged working branch)
- removed the .

Reply:
No link is not correct. It is currently:

https://github.com/ably/jwt-auth-node-tutorial

It should be:

https://github.com/ably/tutorials/tree/jwt-authentication-nodejs

To align with the other links in the tutorial.

Reply:
sorry, it got my wires crossed. fixing this now.